package.json

```diff
@@ -59,6 +59,7 @@
     "redux": "^4.0.5",
     "redux-observable": "^1.2.0",
     "rxjs": "^6.5.3",
+    "uuid": "^9.0.0",
```

Comment:
Would it be possible to use `@sanity/uuid` here? (Which is already included as a dependency in `@sanity/core` and `@sanity/desk-tool`